share/site/duckduckgo/base.tx

```diff
@@ -54,7 +54,6 @@
 		<: } if $copyright_footer { :> 
 			<: include "footer_copyright.tx" :>
 		<: } :>
-		<: if !$no_js { include "footer.tx" } :>
```

Comment:
The footer.tx template was only being used to call the `nip` function, so we no longer need it.